src/main/webapp/placeGuides/PlaceGuideOnList.js

```diff
@@ -36,6 +37,44 @@ class PlaceGuideOnList {
     PlaceGuideOnList.close(placeGuideId);
   }
 
+  static expand(placeGuideId) {
+    const divId = 'placeGuideOnList-' + '{' + placeGuideId + '}';
+    const placeGuideDiv = document.getElementById(divId);
+    placeGuideDiv.querySelectorAll('.folded-placeGuide')[0]
+        .style.display = 'none';
+    placeGuideDiv.querySelectorAll('.card-placeGuide')[0]
+        .style.display = 'block';
+    placeGuideDiv.style.padding = '0px';
+  }
+
+  static getBlobSrc(blobKey) {
+    const src = new URL('/serve-blob', document.URL);
+    src.searchParams.append('blob-key', blobKey);
+    return src;
+  }
+
+  static generateQueryString(placeGuideProperties) {
+    const esc = encodeURIComponent;
```

Comment:
What is "esc"?

Reply:
Not my code, but I assume Dennis used it to make the code shorter. I removed it now.